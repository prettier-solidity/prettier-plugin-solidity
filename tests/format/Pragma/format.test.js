runFormatTest(import.meta, ['slang'], { compiler: '0.8.0' });
