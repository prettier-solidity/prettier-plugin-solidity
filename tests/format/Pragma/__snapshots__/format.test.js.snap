// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pragma.sol format 1`] = `
====================================options=====================================
parsers: ["slang-solidity"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
pragma solidity >=0.4.21   <0.6.0;
pragma solidity >=0.4.21 <=0.6.0    ;
pragma solidity    >0.4.21 <0.6.0;
pragma solidity >0.4.21 <=0.6.0;
pragma solidity >=0.4.21 <0.6.0;
pragma solidity   ^ 0.4.21  ;
pragma solidity  ~ 0.4.21  ;
pragma solidity   ^0.4.0   ||   ^0.5.0   ||   ^0.6.0  ;
pragma solidity   0.5.0   -   0.6.0     ;
pragma   experimental     ABIEncoderV2;
pragma       abicoder     v2;

=====================================output=====================================
pragma solidity >=0.4.21 <0.6.0;
pragma solidity >=0.4.21 <=0.6.0;
pragma solidity >0.4.21 <0.6.0;
pragma solidity >0.4.21 <=0.6.0;
pragma solidity >=0.4.21 <0.6.0;
pragma solidity ^0.4.21;
pragma solidity ~0.4.21;
pragma solidity ^0.4.0 || ^0.5.0 || ^0.6.0;
pragma solidity 0.5.0 - 0.6.0;
pragma experimental ABIEncoderV2;
pragma abicoder v2;

================================================================================
`;
